cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1074,6 +1074,26 @@
       <description>Security realm used for authentication</description>
   </property>
 
+  <!--
+      Security configurations for Kerberos authentication
+  -->
+  <property>
+    <name>cdap.master.kerberos.keytab</name>
+    <value></value>
+    <description>The full path to the Kerberos keytab file containing the CDAP Master's
+    credentials.</description>
+  </property>
+
+  <property>
+    <name>cdap.master.kerberos.principal</name>
+    <value></value>
```

Comment:
Anything wrong with populating this with the example? `cdap/_HOST@EXAMPLE.COM` matches the Hadoop default settings.